invenio_circulation/views.py

```diff
@@ -137,3 +141,65 @@ def post(self, pid, record, action, **kwargs):
             pid, record, HTTP_CODES['accepted'],
             links_factory=self.links_factory
         )
+
+
+def build_blueprint_with_state(app):
+    """Item circulation state blueprint."""
+    blueprint = Blueprint(
+        'invenio_circulation_state',
```

Comment:
Same about naming, similar to the others,`invenio_circulation_item_circulation_state`? (but the duplication of `circulation` is ugly :()